File: Coding/Python/Set3/ncr.py
def factorial(num):
    result = 1
    for x in range(num, 1, -1):
        result *= x
    return result


def ncr(n, r):
    return factorial(n) // (factorial(n - r) * factorial(r))


print(ncr(5, 2))
print(ncr(3, 1))

File: Coding/JavaScript/Set3/ncr.js
console.log(ncr(3,1))

function ncr(n , r){
    return factorial(n)/ (factorial(n-r) * factorial(r))
}

function factorial(n){
    let result = 1;
    for(let i=2;i<=n;i++){
        result *= i
    }
    return result
}